ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -44,20 +57,23 @@ trap {
 #
 
 # Download AzLab module file, import, and then delete the file
+if (-not (Get-Module -ListAvailable -Name 'Az')) {
+    Import-Module Az -Force 
```

Comment:
I think Import-Module isn't sufficient here, if it's not already installed, import-module won't install it.  (Can use Install-Module instead).

Instead of doing that - it might be better to just write an error message and exist the script...  Installing modules (depending on the user's powershell configuration) might require admin permissions...  So it's better if we just tell them to install the Azure PowerShell modules (please run "Install-Module -Name Az" in an elevated powershell window and try again)

Reply:
I just saw the instructions below in the readme - so the user is already told to install the module!  So you should have code that looks like this:

```powershell
if ((Get-Module -ListAvailable -Name 'Az.Accounts') -and
    (Get-Module -ListAvailable -Name 'Az.Resources') -and
    (Get-Module -ListAvailable -Name 'Az.LabServices')) {
        Import-Module -Name Az.Accounts
        Import-Module -Name Az.Resources
        Import-Module -Name Az.LabServices
}
else {
    Write-Error "Unable to run script, Az modules are missing.  Please install them via 'Install-Module -Name Az -Force' from an elevated command prompt"
}
```